### PR TITLE
[Bug 18585] Ensure Dictionary traps cmd+A

### PR DESCRIPTION
**Pull request by livecodepanos**

### CODE REVIEW

File: Toolset/palettes/dictionary/behaviors/revdictionarybehavior.livecodescript [hunk 1]
```diff
@@ -93,7 +93,7 @@ end showUpgradeOptions
 
 # bug 17819 enable cmd+c in dictionary
 on commandKeyDown pWhich
-   if pWhich is not "C" then 
+   if pWhich is not "C" and pWhich is not "A" then
```
Comment (livecodeali) [inline Toolset/palettes/dictionary/behaviors/revdictionarybehavior.livecodescript]: Could we make this a switch as the logic is quite confusing, and comment the code, something like

switch pWhich 
case "A"
-- trap Cmd + A for select all
case "C"
-- trap Cmd + C for copy
break
case "W"
close me
break
default
pass commandKeyDown
end switch
```

Comment (livecodepanos) [reply Toolset/palettes/dictionary/behaviors/revdictionarybehavior.livecodescript]: Good idea :)

**Comment by livecodeali** [comment]
@livecode-vulcan review ok fd57957

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok fd5795718d1058bc390b7d52efca7812facca1e3

**Pull request merged**